pkg/operation/cloudbotanist/gcpbotanist/controlplane.go

```diff
@@ -40,10 +41,23 @@ func (b *GCPBotanist) GenerateCloudProviderConfig() (string, error) {
 		networkName = b.Shoot.SeedNamespace
 	}
 
+	var (
+		subnetID       = "subnet_internal"
+		subNetworkName = ""
+	)
+	tf, err := b.NewShootTerraformer(common.TerraformerPurposeInfra)
+	if err != nil {
+		return "", err
+	}
+	stateVariables, err := tf.GetStateOutputVariables(subnetID)
```

Comment:
`GetStateOutputVariables` may return a `variablesNotFoundError` (see [implementation](https://github.com/gardener/gardener/blob/c8b2d9d4ad4edfff8f89ba76199b1c8628efa4ab/pkg/operation/terraformer/state.go#L47)). Please differentiate between this error and all others.